test/pspm_trim_test.m

```diff
@@ -1,6 +1,6 @@
 classdef pspm_trim_test < matlab.unittest.TestCase
 
```

Comment:
would this blank line interfer with the docunentation generation?

Reply:
> would this blank line interfer with the docunentation generation?

I do not quite remember... however I have sorted out all these blank lines in the code.

Reply:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/3895146/165378576-43802f4c-f369-477a-9ae3-0049a244504f.png">
<img width="688" alt="image" src="https://user-images.githubusercontent.com/3895146/165378598-5b619f5c-283d-4d44-94a1-5ee480df95c1.png">
Currently they are all converted into UTF-8 now.